upstream/solr6/local/92-init-solr.sh

```diff
@@ -148,10 +149,15 @@ function makeConfigs {
                 setOption 'alfresco.corePoolSize' "${ALFRESCO_CORE_POOL_SIZE:-8}" "$CONFIG_FILE_CORE"
                 setOption 'alfresco.doPermissionChecks' "${ALFRESCO_DO_PERMISSION_CHECKS:-true}" "$CONFIG_FILE_CORE"
                 setOption 'solr.suggester.enabled' "$ALFRESCO_SOLR_SUGGESTER_ENABLED" "$CONFIG_FILE_CORE"
-                # SOLR_BACKUP_DIR only useful for ASS v2.* and up
-                if [ $SOLR_VERSION_MAJOR != 1 ]
+                # SOLR_BACKUP_DIR only useful for ASS v2.0.2* and up
+                if [ $SOLR_VERSION_MAJOR = 2 -a $SOLR_VERSION_REV != 0 -a $SOLR_VERSION_REV != 1 ]
```

Comment:
shouldn't this test als include major versions >2 ?

Reply:
Another opinion: can we just always set the `solr.backup.dir` option?

If it is not supported by a version of Solr, it is just ignored, right?
And that is easier to do than a complex condition to check the version number.

Reply:
> shouldn't this test als include major versions >2 ?

Version 3 is elasticsearch. It is not clear that Solr will get new major versions.

Reply:
It is indeed ignored. 

Although I don't really like it to have a property in solrcore.properties that does not function. It might be confusing.